auto_scan.py

```diff
@@ -56,8 +56,13 @@ def build_container_assessment_cache(lw_client, start_time, end_time):
         repository = scanned_container['IMAGE_REPO']
         image_id = scanned_container['IMAGE_ID']
         # tags = scanned_container['IMAGE_TAGS']
-
-        qualified_repo = f'{registry}/{repository}'.lstrip('/')
+        
+        # Images scanned by the inline already contains the registry name
+        if repository.startswith(registry):
+            qualified_repo = f'{repository}'.lstrip('/')
+        else:
+            qualified_repo = f'{registry}/{repository}'.lstrip('/')
+        
```

Comment:
🙌 Thank you so much for the contribution!  It looks like this might have been a change in `v4.90` of the platform that I was unaware of - previously, containers scanned with the Inline Scanner would return an empty registry value, which is why the `lstrip('/')` was required after concatenation.

This is extremely nit picky, but can we move the lstrip out of the conditional for readability?

```suggestion
        # Images scanned by the inline already contains the registry name
        if repository.startswith(registry):
            qualified_repo = f'{repository}'
        else:
            qualified_repo = f'{registry}/{repository}'
        qualified_repo = qualified_repo.lstrip('/')
```